common/src/main/java/com/wynntils/features/user/CustomNametagRendererFeature.java

```diff
@@ -7,33 +7,117 @@
 import com.wynntils.core.config.Config;
 import com.wynntils.core.features.UserFeature;
 import com.wynntils.core.managers.Model;
+import com.wynntils.core.webapi.WebManager;
+import com.wynntils.core.webapi.profiles.item.ItemProfile;
 import com.wynntils.mc.event.NametagRenderEvent;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.mc.utils.McUtils;
 import com.wynntils.wynn.model.UserInfoModel;
 import com.wynntils.wynn.objects.account.AccountType;
+import com.wynntils.wynn.utils.WynnPlayerUtils;
 import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.player.LocalPlayer;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.entity.projectile.ProjectileUtil;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.minecraft.world.phys.AABB;
+import net.minecraft.world.phys.EntityHitResult;
+import net.minecraft.world.phys.Vec3;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public class CustomNametagRendererFeature extends UserFeature {
+    private static final float RAYCAST_RANGE = 5f;
+
     @Config
     public boolean hideAllNametags = false;
 
     @Config
     public boolean showAccountType = true;
 
+    @Config
+    public boolean showGearOnHover = true;
```

Comment:
Is this so popular that it should be true by default? I think I'd mostly be annoyed by it.

Reply:
Oh that might be the case. It might be off by default in legacy too.

Reply:
I think it should be true by default. You typically don't encounter that many players on your server in the gameplay